src/newUi.scss

```diff
@@ -934,6 +934,7 @@ nav {
     width: fit-content !important;
     background-color: var(--secondary-color);
     z-index: 1000;
+    padding: 5px 10px;
```

Comment:
chore: This property redefined a `padding` that is already present in this class. We should have a single definition of each property

Reply:
First property removed. Waiting for review.